openmeetings-server/src/site/markdown/LdapAndADS.md

```diff
@@ -0,0 +1,238 @@
+    ldap_user_attr_town=l
+    ldap_user_attr_phone=telephoneNumber
+    # optional attribute for user picture
+    #ldap_user_attr_picture=
+    ldap_group_attr=memberOf
+    
+    # optional, absolute URL will be used as user picture if #ldap_user_attr_picture will be empty
+    #ldap_user_picture_uri=picture_uri
+    
+    # Ldap ignore upper/lower case, convert all input to lower case
+    ldap_use_lower_case=false
+    
+    # Ldap import query, this query should retrieve all LDAP users
+    ldap_import_query=(objectClass=inetOrgPerson)
+    
+    ``` 
+    
+    NOTE: Directory Servers are individual for every organization, and
+    might use different attribute names than the ones listed in the
+    example config. Please obtain the information about the structure
+    and attribute names of the entries and adjust the config file
+    accordingly.
+
+    It is important to configure LDAP_SEARCH_PATH so that it points to the
+    node of the LDAP directory server that contains the users.
+
+    If there are multiple search paths you can either try to define a
+    search path that points to multiple nodes or define multiple LDAP
+    configurations in the LDAP administration pane of OpenMeetings.
+
+### 3. Configuring OpenMeetings to use the ldap config file
+
+The final step binds the config file of the previous section to
+OpenMeetings:
+
+-   Log into OpenMeetings as Administrator
+
+-   Open Administration->LDAP
+
+    <img src="../resources/images/ldap_screen.png"></img>
+
+-   In the right pane of the main window labeled "LDAP Config" set
+    the Name of the LDAP DB which you want to appear in the popup
+    menu right of the "Domain" label of OpenMeeting's login
+    screen. Any Name can be set here.
+
+-   Check the "Enabled" checkbox to make the LDAP DB available on the
+    login screen.
+
+-   Fill in the field "Config file name" with the file name of the
+    config file from the previous section of this manual (just the
+    file name, without it's path).
+
+-   Save the Configuration by clicking in the save icon above the
+    "LDAP Config" label and load this setting by clicking in the
+    reload icon right of the save icon.
+
+-   The table in the left pane of OpenMeeting's LDAP administration
+    window should contain an entry with the ID (autoassigned by
+    OpenMeetings), name and config file.
+
+Test the settings by logging out of Openmeetings and logging back in
+with a user and password of the LDAP DB. Make sure you select the
+configured name of the LDAP DB in the "Domain" popup menu of the Login
+screen before pressing the "Sign in" button. If login doesn't work,
+see the section [Troubleshooting](#ldap-troubleshooting)
+
+## Additional (optional) steps
+
+Once login with users from the LDAP DB works, all users of that
+database can get imported into OpenMeetings internal table of users
+using the command line tool "admin.sh":
+
+-   First test with:
+    
+    `$OM_HOME/admin.sh -l -d1 --print-only`
```

Comment:
this is the place search parameters will work

Reply:
Hmm, what do you mean by that? Which search parameters? How are they submitted?

Reply:
`ldap_admin_dn, ldap_passwd, ldap_search_base, ldap_search_query, ldap_search_scope` from config referenced by `d1` ID :)